back/lib/participation_method/base.rb

```diff
@@ -93,11 +82,11 @@ def extra_fields_category_translation_key
       'custom_forms.categories.extra.title'
     end
 
-    def supports_exports?
+    def supports_public_visibility?
       false
     end
 
-    def supports_publication?
+    def supports_exports?
```

Comment:
What does this mean? I think all participation methods support exports, no?

Reply:
I agree this is confusing. It was already there so I just left it untouched.

It's used to generate an xlsx file with one sheet tab per phase, only including native survey, proposals, ideation and voting phases.